geoips/plugins/modules/readers/utils/geostationary_geolocation.py

```diff
@@ -19,29 +19,30 @@
 from pyresample.geometry import SwathDefinition
 from pyresample.kd_tree import get_neighbour_info  # , get_sample_from_neighbour_info
 
+from geoips.errors import CoverageError
+from geoips.filenames.base_paths import PATHS as gpaths
+
 LOG = logging.getLogger(__name__)
 
 
 try:
     import numexpr as ne
 except ImportError:
     LOG.info(
-        "Failed import numexpr in scifile/readers/abi_ncdf4_reader_new.py. If you need it, install it."
+        "Failed import numexpr in scifile/readers/abi_ncdf4_reader_new.py."
+        "If you need it, install it."
     )
 
-from geoips.errors import CoverageError
-from geoips.filenames.base_paths import PATHS as gpaths
-
 
 nprocs = 6
 
 try:
     ne.set_num_threads(nprocs)
 except Exception:
     LOG.info(
-        "Failed numexpr.set_num_threads in {}. If numexpr is not installed and you need it, install it.".format(
-            __file__
-        )
+        "Failed numexpr.set_num_threads in {}. If numexpr is not installed and you "
+        "need it, install it.",
+        __file__,
     )
 
```

Comment:
Hmm, is this actually correct?  Using {} with LOG replacements?  It was wrong before too if not...

Reply:
You're right, I missed that one. Fixed it.